UnityProject/Assets/Scripts/Player/ItemLightControl.cs

```diff
@@ -87,45 +92,32 @@ private void LightConsistency()
 			: new Vector3(Size, Size, Size);
 	}
 
-	public void OnInventoryMoveServer(InventoryMove info)
+	public bool IsValidSetup(RegisterPlayer player)
 	{
-		//was it transferred from a player's visible inventory?
-		if ((info.FromPlayer == null || info.ToPlayer == null) && LightEmission != null)
+		if (player == null) return false;
+		//more logic stuff here
```

Comment:
is this missing something?

Reply:
well there would be a cheque for the compatible slot however cyborg lights are in body storage so idk how that would work and they worked previously